### PR TITLE
Couchbase :: Fixed GRD-61145

### PR DESCRIPTION
**Pull request by ldmahale108**
Signed-off-by: Laxmikant Mahale <Laxmikant.Mahale@ibm.com>

### CODE REVIEW

File: filter-plugin/logstash-filter-couchbasedb-guardium/CouchbasedbOverFilebeatPackage/CouchbaseDB/filter.conf [hunk 1]
```diff
@@ -3,7 +3,7 @@ filter  {
     
 	mutate {
       add_field => { "serverIP" => "%{[@metadata][ip_address]}" }
-      add_field => { "serverHostname" => "%{[agent][hostname]}" }
+      add_field => { "serverHostname" => "%{[host][hostname]}" }
```
Comment (RefaelAdi) [inline filter-plugin/logstash-filter-couchbasedb-guardium/CouchbasedbOverFilebeatPackage/CouchbaseDB/filter.conf:6]: it's should be add_field => { "serverHostname" => "%{[host][name]}" }

File: filter-plugin/logstash-filter-couchbasedb-guardium/couchbasedbFilebeat.conf [hunk 2]
```diff
@@ -10,7 +10,7 @@ filter  {
 		
 		mutate {
 			add_field => { "serverIP" => "%{[@metadata][ip_address]}" }
-			add_field => { "serverHostname" => "%{[agent][hostname]}" }
+			add_field => { "serverHostname" => "%{[host][hostname]}" }
```
Comment (RefaelAdi) [inline filter-plugin/logstash-filter-couchbasedb-guardium/couchbasedbFilebeat.conf:13]: also here.

**Review by RefaelAdi — CHANGES_REQUESTED** [review]

File: filter-plugin/logstash-filter-couchbasedb-guardium/CouchbasedbOverFilebeatPackage/CouchbaseDB/filter.conf (hunk 1 above)
Comment (ldmahale108) [reply filter-plugin/logstash-filter-couchbasedb-guardium/CouchbasedbOverFilebeatPackage/CouchbaseDB/filter.conf:6]: `add_field => { "serverHostname" => "%{[host][name]}" }`
also works and `%{[host][hostname]}` is a alternative way(solution).
**QA team has verified this fixes**

**Review by ofer-haim — CHANGES_REQUESTED** [review]
Please wait to hear from QA team about this hostname issue.

**Comment by ldmahale108** [comment]
since this is getting fixed as a part of INS-21528, closing this

**Pull request closed without merging**